lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -37,7 +37,14 @@ defmodule ExUnit.CaptureLog do
   Returns the binary which is the captured output.
 
   This function mutes the `:console` backend
-  and captures any log messages sent to Logger.
+  and captures any log messages sent to Logger from the calling
+  processes. It is possible to ensure explicit log messages from other
+  processes are captured by waiting for their exit or monitor
+  signal. This does not guarantee to capture VM generated log messages
+  when a process is spawned using a low level `Kernel` spawn function (e.g.
+  `Kernel.spawn/1`) does not catch an exception. A `Task`, or other OTP
+  process, will send explicit logs that are sent before its exit or monitor
+  signal and will not cause VM generated log messages.
```

Comment:
It's better, but I think the "does not catch an exception" part is a bit confusing.  This spawned process does not catch an exception, but I don't think it's what you mean:

``` elixir
spawn(fn -> :ok end)
```

Maybe "raises an unhandled exception" or just "crashes" or something similar would be clearer?